crates/jstz_cli/src/sandbox/daemon.rs

```diff
@@ -254,6 +257,26 @@ pub fn main(cfg: &mut Config) -> Result<()> {
     println!("Saving sandbox config");
     cfg.save()?;
 
+    // Run jstz node
+    let local = tokio::task::LocalSet::new();
+    let _ = local
+        .run_until(async {
+            tokio::task::spawn_local(async {
+                println!("Jstz node started 🎉");
+                let _ = run_node(
+                    DEFAULT_ROLLUP_NODE_RPC_ADDR.to_string(),
+                    DEFAULT_ROLLUP_RPC_PORT,
+                    None,
+                    DEFAULT_KERNEL_FILE_PATH.to_string(),
+                    false,
+                )
+                .await;
+            })
+            .await
+            .expect("Local task panicked");
+        })
+        .await;
```

Comment:
Jstz node apparently cannot be moved between threads because it involves some non-thread safe behaviour, so  LocalSet is essentially creating a context for it so that it can be run.